gems/sorbet/bin/srb-rbi

```diff
@@ -210,10 +210,8 @@ If instead you want to explore your files locally, here are some things to try:
       make_step(Sorbet::Private::TodoRBI)
     when 'suggest-typed'
       make_step(Sorbet::Private::SuggestTyped)
-
-    when 'find-gem-rbis', "#{ENV['HOME']}/.cache/sorbet/gem-rbis", "#{ENV['HOME']}/.cache/sorbet/gem-rbis/"
+    when 'find-gem-rbis', Sorbet::Private::FindGemRBIs.cache_dir
```

Comment:
the prior version was agnostic of when path is `/` terminated or not, isn't this lost with this change?